toqito/measurements/pretty_good_measurement.py

```diff
@@ -0,0 +1,67 @@
+"""Compute the set of pretty good measurements from an ensemble."""
+import numpy as np
+import scipy
+
+from toqito.matrix_ops import vector_to_density_matrix
+
+
+def pretty_good_measurement(states: list[np.ndarray], probs: list[float] | None = None) -> list[np.ndarray]:
+    r"""Return the set of pretty good measurements from a set of vectors and corresponding probabilities.
+
+    This computes the "pretty good measurement" as initially defined in :cite:`Hughston_1993_Complete`.
```

Comment:
It might be better to define what "pretty good measurement" is in the docstring. Looking through the docs, we seem to consistently define the math behind the function. 

For example, https://toqito.readthedocs.io/en/latest/autoapi/channel_metrics/channel_fidelity/index.html#channel_metrics.channel_fidelity.channel_fidelity

Reply:
Yes, absolutely. This has been updated (and has also been done for the PBM as well). Good call!